lib/server/server.js

```diff
@@ -133,7 +132,7 @@ function execute(port) {
 
   app.get(/docs\/.*html$/, (req, res, next) => {
     let url = req.path.toString().replace(siteConfig.baseUrl, '');
-
+    // let url = 'docs/en/installation.html'
```

Comment:
We may have to consider the default language here. But this may be ok for now. I will have to check.